R/tf_estimator.R

```diff
@@ -371,7 +371,7 @@ export_savedmodel.tf_estimator <- function(object,
       features <- list()
       for (feature in feature_columns_spec) {
         # first dimension is variable since it's required by cloudml-like interfaces to push multiple instances
-        features[[feature$name]] <- tf$placeholder(shape = shape(NULL, feature$shape), dtype = feature$dtype)
+        features[[feature$name]] <- tf$placeholder_with_default(shape = shape(NULL, feature$shape), dtype = feature$dtype)
```

Comment:
We need to specify the default `input` and not `dtype`: https://www.tensorflow.org/api_docs/python/tf/placeholder_with_default